### ISSUE TITLE
Kurzanleitung verbessern

### ISSUE THREAD

**Issue by alexplusde** [Sioweb/redaxo_assets#21]
Hallo nochmal,

irgendwie sind doch ein paar Sachen unklar in der Verwendung, bei mir werden bspw. keine JavaScript-Dateien erkannt und auch die Generierung der SCSS-Dateien läuft irgendwie schief.

Können wir da evtl. mal skypen? Ich würde gerne im Gegenzug die Anleitung erweitern.

**Comment by Sioweb** [comment]
Klar, abends 17:30 bin ich meist erreichbar: Sioweb in Skype. SASS wird noch nicht korrekt unterstützt, da mir noch keiner sagen konnte was hier gebraucht wird.

**Comment by alexplusde** [comment]
Stark! Ich mach mir mal ein paar Gedanken und Notizen.

**Comment by Sioweb** [comment]
Ho! JFI heute hab ich keine Zeit mehr, morgen oder Samstags morgens :)

**Comment by alexplusde** [comment]
Wird bei mir auch eher nächste Woche ;)
